packages/react-components/src/organisms/LeadershipMembershipTable.tsx

```diff
@@ -59,43 +78,183 @@ export type TeamMetric = {
   memberCount: number;
   previousMemberCount: number;
 };
-interface LeadershipMembershipTableProps {
-  data: TeamMetric[];
-}
+type LeadershipMembershipTableProps = Pick<
+  ComponentProps<typeof LeadershipPageBody>,
+  | 'data'
+  | 'metric'
+  | 'sort'
+  | 'setSort'
+  | 'sortingDirection'
+  | 'setSortingDirection'
+>;
 
 const LeadershipMembershipTable: React.FC<LeadershipMembershipTableProps> = ({
   data,
-}) => (
-  <Card padding={false}>
-    <div css={container}>
-      <div css={[rowStyles, gridTitleStyles]}>
-        <span css={titleStyles}>Team</span>
-        <span css={titleStyles}>Currently in a leadership role</span>
-        <span css={titleStyles}>Previously in a leadership role</span>
+  metric,
+  sort,
+  setSort,
+  sortingDirection,
+  setSortingDirection,
+}) => {
+  const indexPrefix = metric === 'working-group' ? 'wg' : 'ig';
+  return (
+    <Card padding={false}>
+      <div css={container}>
+        <div css={[rowStyles, gridTitleStyles]}>
+          <span css={titleStyles}>
+            Team
+            <button
+              css={buttonStyles}
+              onClick={() => {
+                if (sortingDirection.team === 'asc') {
+                  setSort('team_desc');
+                  setSortingDirection({
```

Comment:
I noticed that when you change to a different sorting order the previous order icon remains the same, which could be confusing. I've attached a video. Maybe this should set to the initialSortingDirection except the one being selected?

https://github.com/yldio/asap-hub/assets/25244556/4fe1a742-25ba-4854-af41-ed9a12ccbebf

Reply:
nice! it makes sense. I've added it here [setSortingDirection sets to initialSortingDirection + icon direction](https://github.com/yldio/asap-hub/pull/4254/commits/5ed6bf0f2146ec9ec645e3258ce7033fb84f871a)